docs/topics/request-response.rst

```diff
@@ -219,6 +231,22 @@ different fields from different pages::
         item['other_url'] = response.url
         yield item
 
+::
+
+    def parse_page1(self, response):
+        item = MyItem()
+        item['main_url'] = response.url
+        request = scrapy.Request("http://www.example.com/some_page.html",
+                                 callback=self.parse_page2)
+        request.cb_kwargs['item'] = item
```

Comment:
I see you followed the same pattern as the existing example about `meta`, however I wonder if the example should use the constructor parameter instead of using the object property. Or at least both should be covered (one field set on each).

Reply:
Updated, thanks.